src/lang/Messages.properties

```diff
@@ -1274,6 +1293,7 @@ http.panel.component.all.tooltip = Combined display for header and body
 http.panel.component.split.body = Body: 
 http.panel.component.split.header = Header: 
 http.panel.component.split.tooltip = Split display for header and body
+http.panel.name = HTTP Panels Extension
```

Comment:
Maybe call it "Combined HTTP Panels Extension"? This extension is the one that shows the header and body in single/combined view.